docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.3'
+
+services:
+  almadar:
+    restart: unless-stopped
```

Comment:
:themoreyouknow:

Reply:
learned it from playing with the sentry setup last week!